README.md

```diff
@@ -0,0 +1,61 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Description the project.
+    This is a very first project to say Hello Microverse and esnure using of linters!
+
+## Built With
+
+- html,css
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] [OPTIONAL] I would advise that you include contents to these empty sections or remove it completely. This will make your README more professional.

Reply:
thanks very much for your suggestions.
i `v taken it into account and removed them